packages/@react-spectrum/progress/docs/ProgressBar.mdx

```diff
@@ -0,0 +1,125 @@
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue`.
+
+```tsx example
+<ProgressBar label="Loading…" minValue={50} maxValue={150} value={100} />
+```
+
+Values are formatted as a percentage by default, but can be modified by using the `formatOptions` for different formats.
+`formatOptions` is compatible with the option parameter of [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) and is applied to the current locale.
+
+```tsx example
+<ProgressBar label="Loading…" formatOptions={{style: 'currency', currency: 'JPY'}} value={60} />
+```
+
+### Indeterminate
+[View guidelines](https://spectrum.adobe.com/page/bar-loader/#Determinate-or-indeterminate)
+
+By default, Progress Bars are determinate. Use a determinate Progress Bar when progress can be calculated against a specific goal.
+
+```tsx example
+<ProgressBar label="Loading…" value={50} />
+```
+
+Use an indeterminate Progress Bars when progress is happening but the time or effort to completion can’t be determined.
+
+```tsx example
+<ProgressBar label="Loading…" isIndeterminate />
+```
+
+### Accessibility
+
+Depending on the visualization being used (i.e. depending on the `showValueLabel` prop), a `label`, `aria-label`, or `aria-labelledby` attribute are required.
+
+```tsx example
+<div>
+  <ProgressBar aria-labelledby="pc-label" showValueLabel={false} value={45} />
+  <label id="pc-label" style={{display: 'block'}}>Loading…</label>
+</div>
+```
+
+If using the over background variant, ensure the background offers enough contrast for the Progress Bar to be legible and meets color contrast guidelines.
+
+### Internationalization
+
+To internationalize a Progress Bar, a localized string should be passed to the `label` prop, `aria-label` prop, or value of the associated `aria-labelledby` element.
+
+For languages that are read right to left (e.g. Hebrew and Arabic), the layout is mirrored for both determinate and indeterminate options.
+
+## Props
+
+<PropTable component={docs.exports.ProgressBar} links={docs.links} />
+
+## Visual Options
+
+### Over background
+[View guidelines](https://spectrum.adobe.com/page/bar-loader/#Over-background)
+
+When a Progress Bar needs to be placed on top of a colored background, use the `overBackground` variant.
+
+```tsx example
+<View backgroundColor="positive" padding="25px">
+  <ProgressBar label="Loading…" variant="overBackground" value={5} />
+</View>
+```
+
+### Labels
+[View guidelines](https://spectrum.adobe.com/page/bar-loader/#Label-or-no-label)
+
+Value labels are shown by default on top, but they can also be moved to the side or hidden as needed.
+
+```tsx example
+<Flex flexDirection="column" maxWidth="250px">
+  <ProgressBar label="Loading…" marginBottom="25px" value={30} />
+  <ProgressBar label="Loading…" marginBottom="25px" labelPosition="side" value={30} />
+  <ProgressBar label="Loading…" showValueLabel={false} value={30} />
```

Comment:
Would this last example be better without the `label="Loading…"`?

Reply:
presumably it also sets aria-label, so i can either convert to that or leave it as is. lmk

Reply:
the label is different from the valuelabel so either label or aria-label will be needed anyway.